python/cuml/decomposition/pca.pyx

```diff
@@ -465,14 +465,22 @@ class PCA(Base):
             Acceptable formats: cuDF DataFrame, NumPy ndarray, Numba device
             ndarray, cuda array interface compliant array like CuPy
 
+        convert_dtype : bool (default = False)
```

Comment:
Assuming this is only conditionally converting when it needs to, I think we should be more clear in the verbiage:

```
When set to True, the inverse_transform method will, when necessary, convert the input to the data which was used to the train the model....
```